pkg/lima/lima.go

```diff
@@ -40,7 +40,7 @@ func GetVMStatus(creator command.LimaCmdCreator, logger flog.Logger, instanceNam
 func toVMStatus(status string, logger flog.Logger) (VMStatus, error) {
 	logger.Debugf("Status of virtual machine: %s", status)
 	switch status {
-	case "":
+	case "", "Nonexistent":
```

Comment:
This method is to translate from upstream state to local package state. the upstream state doesn't have "Nonexistent". It just returns "" to show it is nonexistent.

Reply:
Got you, thanks. Removed it